cmd/docker-app/root.go

```diff
@@ -11,8 +11,7 @@ import (
 )
 
 // rootCmd represents the base command when called without any subcommands
-// FIXME(vdemeester) use command.Cli interface
-func newRootCmd(dockerCli *command.DockerCli) *cobra.Command {
+func newRootCmd(cli command.Cli) *cobra.Command {
```

Comment:
nit: I didn't use `cli` here to not *shadow* the `cli` package (`docker/cli/cli`) if it is used somewhere

Reply:
Ah yes, I realized that once I tried to replace it everywhere else.
I'll revert that.